userspace/libsinsp/filterchecks.cpp

```diff
@@ -109,6 +110,317 @@ static inline bool str_match_start(const std::string& val, size_t len, const cha
 }
 
 #define STR_MATCH(s) str_match_start(val, sizeof (s) -1, s)
+///////////////////////////////////////////////////////////////////////////////
+// sinsp_filter_check_fspath implementation
+///////////////////////////////////////////////////////////////////////////////
+const filtercheck_field_info sinsp_filter_check_fspath_fields[] =
+	{
+		{PT_CHARBUF, EPF_NONE, PF_NA, "fspath.name", "Path for Filesystem-related operation", "For any event type that deals with a filesystem path, the path the file syscall is operating on. This path is always fully resolved, prepending the thread cwd when needed."},
+		{PT_CHARBUF, EPF_NONE, PF_NA, "fspath.nameraw", "Raw path for Filesystem-related operation", "For any event type that deals with a filesystem path, the path the file syscall is operating on. This path is always the path provided to the syscall and may not be fully resolved."},
+		{PT_CHARBUF, EPF_NONE, PF_NA, "fspath.source", "Source path for Filesystem-related operation", "For any event type that deals with a filesystem path, and specifically for a source and target like mv, cp, etc, the source path the file syscall is operating on. This path is always fully resolved, prepending the thread cwd when needed."},
+		{PT_CHARBUF, EPF_NONE, PF_NA, "fspath.sourceraw", "Source path for Filesystem-related operation", "For any event type that deals with a filesystem path, and specifically for a source and target like mv, cp, etc, the source path the file syscall is operating on. This path is always the path provided to the syscall and may not be fully resolved."},
+		{PT_CHARBUF, EPF_NONE, PF_NA, "fspath.target", "Target path for Filesystem-related operation", "For any event type that deals with a filesystem path, and specifically for a target and target like mv, cp, etc, the target path the file syscall is operating on. This path is always fully resolved, prepending the thread cwd when needed."},
+		{PT_CHARBUF, EPF_NONE, PF_NA, "fspath.targetraw", "Target path for Filesystem-related operation", "For any event type that deals with a filesystem path, and specifically for a target and target like mv, cp, etc, the target path the file syscall is operating on. This path is always the path provided to the syscall and may not be fully resolved."},
+};
+
+sinsp_filter_check_fspath::sinsp_filter_check_fspath()
+{
+	m_info.m_name = "file";
```

Comment:
```suggestion
	m_info.m_name = "fspath";
```

Reply:
Yes will fix.

Reply:
Mark let's wait until Monday until we get one more round of feedback from @jasondellaluce around final naming throughout as now perhaps `fs` is winning and then maybe change to `fs.path.name` or `fs.path` or `fs.name` we just need to agree.
Just so you don't have to rename stuff 5 times.

Reply:
Updated to `fs.path.*`.